### PR TITLE
MBM-29: Inital Addition Of Earth Toggle

### PR DESCRIPTION
**Pull request by YonatanWN**

### CODE REVIEW

File: src/main/java/com/mineboundteam/minebound/capabilities/PlayerUtilityToggleProvider.java [hunk 1]
```diff
@@ -53,17 +53,21 @@ public static void UpdatePlayerSync(Player oldPlayer, Player newPlayer){
     }
     public static class UtilityToggle {
         public boolean fire = false;
+        public boolean earth = false;
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/capabilities/PlayerUtilityToggleProvider.java:56]: should this one default to on?

Comment (MattKirchh) [reply src/main/java/com/mineboundteam/minebound/capabilities/PlayerUtilityToggleProvider.java:56]: I actually don't really understand the functionality of Earth Utility from the SRS. Is it like Earth Defensive but with a set mining speed + vein miner?

Comment (YonatanWN) [reply src/main/java/com/mineboundteam/minebound/capabilities/PlayerUtilityToggleProvider.java:56]: > should this one default to on?

Yeah Probably I'd rather have vein miner on initially

Comment (YonatanWN) [reply src/main/java/com/mineboundteam/minebound/capabilities/PlayerUtilityToggleProvider.java:56]: > 

I think just vein miner vs. regular mining

Comment (insertStringHere) [reply src/main/java/com/mineboundteam/minebound/capabilities/PlayerUtilityToggleProvider.java:56]: ooh wait does it include vein miner? alright, in that case nevermind. 

It'll add mining speed and vein miner when breaking a block. Vein miner if the toggle is on, mining speed regardless

Comment (MattKirchh) [reply src/main/java/com/mineboundteam/minebound/capabilities/PlayerUtilityToggleProvider.java:56]: Got it, ok. We don't mention the increase to the mining speed but I'll add that to the SRS.

**Review by MattKirchh — APPROVED** [review]

**Review by aat79 — APPROVED** [review]

**Pull request merged**